Fakebook.Profile/Fakebook.Profile.UnitTests/DomainTests/ProfileModelTests.cs

```diff
@@ -0,0 +1,305 @@
+﻿using System;
+
+using Fakebook.Profile.Domain;
+
+using Xunit;
+
+namespace Fakebook.Profile.UnitTests.DomainTests
+{
+    public class ProfileModelTests
+    {
+        #region EmailTests
+        /// <summary>
+        /// Check that emails can't be null.
+        /// </summary>
+        [Fact]
+        public void EmailCantBeNull()
```

Comment:
the properties in question should actually throw notimplementedexceptions also, so that tests like "can i assign a valid email" properly fail at first.